CHANGES.rst

```diff
@@ -9,6 +9,14 @@ Unreleased
 **Added**
 
 - 301 redirects result in a ``Redirect`` exception.
+- Calls to method ``ScriptAuthorizer.refresh`` are handled by a private method,
+  ``ScriptAuthorizer._refresh_with_retries``. The latter provides the option to retry 
```

Comment:
I don't think it's necessary to mention the private method in the change log. Just that retries are now supported.